### ISSUE TITLE
[COGNITION]: Let's add aria-hidden attributes to the check icons on our confirmation page

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6243]
## Description

The checkboxes on our /confirmation page are tab stops but do not announce anything to screen readers. I'd like to add an `aria-hidden="true" so we can remove this unexpected tab stop.
Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* [ ] As a screen reader user, I'd like to not hear the "blank" read out when I'm navigating through with virtual cursor

## Environment

* Windows 10
* Chrome/IE11
* JAWS
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-appointment/review

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-02-26 at 3 02 15 PM](https://user-images.githubusercontent.com/934879/75387817-827a6c80-58a9-11ea-932d-a23ed8684cfd.png)
